index.bs

```diff
@@ -1206,6 +1207,26 @@ the application will receive the number of streams it anticipates.
            1. [=Resolve=] |p| with |stats|.
      1. Return |p|.
 
+: <dfn for="WebTransport" method>deriveKeyingExporter(BufferSource label, optional BufferSource context)</dfn>
```

Comment:
Is this the right name.  `exportX` would be better, I think.

Reply:
Updated the name, but I'll leave the type change determination to @vasilvv based on the discussion in the issue.